lang/fr.po

```diff
@@ -2,8 +2,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Performous\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2017-08-30 22:35+0200\n"
-"PO-Revision-Date: 2017-08-30 22:35+0200\n"
+"POT-Creation-Date: 2020-03-31 21:55+0200\n"
+"PO-Revision-Date: 2020-03-31 22:15+0200\n"
 "Last-Translator: Arjan Speiard | Asgard Sings! <arjan.spieard@gmail.com>\n"
```

Comment:
@OznOg give yourself some credit ;)

Reply:
Also add yourself to the translator list (for french)

Reply:
Ok, You speak very good french BTW :)